src/licensedcode/data/rules/gpl-3.0-plus_22.yml

```diff
@@ -0,0 +1,2 @@
+licenses:
+    - gpl-3.0-plus

```

Comment:
Why would this be GPL 3.0?
and further more GPL 3.0 or later?
I did not see any version mentioned in the rule.